### PR TITLE
Add minimal prettier configuration

### PR DESCRIPTION
**Pull request by nicojust**
## Description
Set the minimal required prettier configuration. Added the appropriate packages as dev dependencies. Updated eslint to the latest version.

## Related issue
#59

## How can this be tested?
Use single quotes and save your changes. The single quotes should be converted to double quotes.

## Motivation and context
Ease the develoment by automatically fixing small and tedious things.

## Additional context
Enable the `formatOnSave` setting in vscode.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
+    node: true,
+    jest: true
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "tsconfig.eslint.json",
+    sourceType: "module"
+  },
+  plugins: ["@typescript-eslint", "@typescript-eslint/tslint", "prettier"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:prettier/recommended"],
```
[the remark this answers is not in the document]
Comment (nicojust) [reply .eslintrc.js]: I agree with you. The problem is that prettier doesn't have the option right now to support this format without using some kind of workaround. What do you think would be the best way to move forward?

See https://github.com/prettier/prettier-vscode/issues/352 for an ongoing discussion.

[the remark this answers is not in the document]
Comment (pascal-zarrad) [reply .eslintrc.js]: I think we should look out for a workaround to fix this problem.
The current array style of prettier is okay for small arrays, but awfull for arrays like this "extends".
We need something like the ESLint rule "array-element-newline" with the configuration parameter "consistent" or "always"

[the remark this answers is not in the document]
Comment (nicojust) [reply .eslintrc.js]: Unfortunately ther is no such option yet. The workaround would be to use a comment.

```javascript
  extends: [
    //,
    "eslint:recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:prettier/recommended"
  ],
```

[the remark this answers is not in the document]
Comment (pascal-zarrad) [reply .eslintrc.js]: This is definitly not a great workaround, we should try to find some better way to deal with this.

**Comment by pascal-zarrad** [comment]
I will close this PR as prettier does not fit our needs.
Maybe we can try out ESLint's auto fix feature or find some better solution.

Let's continue the discussion in #59

**Pull request closed without merging**